src/crawler/ipfs.js

```diff
@@ -0,0 +1,50 @@
+const ChluIPFS = require('chlu-ipfs-support')
+
+let chluIpfsSingleton = null
+
+async function getChluIPFS() {
```

Comment:
This function should be removed. `chluIpfs` is available from the `API Publisher` class as `this.chluIpfs`. We must reuse that instance